docs/translations.txt

```diff
@@ -0,0 +1,115 @@
+Translations and anaconda Development
+24-Feb-2015
+by: David Cantrell <dcantrell@redhat.com>
+    Brian C. Lane <bcl@redhat.com>
+-----------------------------------------------------------------------------
+
+Anaconda, as of version 23.1, is using https://fedora.zanata.com for
+translations. You will need an account in order to pull translations until bug
+1172618 (anonymous pull requests) has been resolved.
+
```

Comment:
A URL of the bugreport could be useful here. Or at least add 'rhbz#' so that it is clear that it is a bugzilla.redhat.com bug report.